doc/1.4/language.md

```diff
@@ -1435,6 +1455,19 @@ local uint32 v = bf;
 
 </dd></dl>
 
+### Serializable types
+_Serializable types_ are types that the DML compiler knows how to serialize and
+deserialize for the purposes of checkpointing. This is important for the use of
+[`saved` variables](#saved-variables) and the [`after`
+statement](#after-statements).
+
+All primitive non-pointer data types (integers, floating-point types, booleans,
+etc.) are considered serializable, as is any struct, layout, or array type
+consisting entirely of serializable types. Any pointer type is not considered
```

Comment:
I'm genuinely unsure if the plural/singular mix "*are* considered, as *is*" is good English.

Reply:
Couldn't find anything about it from cursory research, but I see no problem with it. E.g. "They are wrong, as is he."

Reply:
I agree that there's nothing *wrong* with it. Question is if e.g. s/All..are/Any..is/ would be *better*. Don't know.